docs/valunteers.md

```diff
@@ -1,14 +1,14 @@
-# Valunteers
+# Volunteers
 
 ## Tech
 
-- [DevHelp - Tech Valunteers Whatsapp](<https://chat.whatsapp.com/GLmj3FDWiCW1RWDRnL79UV>)
+- [DevHelp - Tech Volunteers Whatsapp](<https://chat.whatsapp.com/GLmj3FDWiCW1RWDRnL79UV>)
 - מערך המתנדבים של יוצאי יחידות המודיעין
   - מיתר 0543047255
-- [General Form for Tech Valunteers](<https://forms.gle/wVB2zNKRZSUhQb79A>)
+- [General Form for Tech Volunteers](<https://forms.gle/wVB2zNKRZSUhQb79A>)
 - [General Discord to manage initiatives](<https://discord.gg/3Jg6us3b>)
 
-## Connecting between valunteers and help seekers
+## Connecting between volunteers and help seekers
 
 - [iron-swords-valunteers-kris](<https://iron-swords-valunteers-kris.co.il>)
```

Comment:
i'm not sure if the domain has already been set, so i didn't change the url here.